app/cdap/components/ConnectionList/Components/CustomTooltip/index.tsx

```diff
@@ -17,13 +17,31 @@
 import { Box, Tooltip, TooltipProps } from '@material-ui/core';
 import * as React from 'react';
 import { useStyles } from './styles';
+import styled from 'styled-components';
+import T from 'i18n-react';
+
+const TooltipStyle = styled(Box)`
+  width: 100%;
+`;
+
+const TooltipUnsetStyle = styled(Box)`
+  width: unset;
+`;
+
+const getWrapperComponent = (tooltipText) => {
+  return tooltipText ? TooltipUnsetStyle : TooltipStyle;
+};
 
 export default function CustomTooltip(props: TooltipProps) {
   const classes = useStyles();
+  const TooltipWrapperComponent = getWrapperComponent(
+    props.title ===
```

Comment:
What is the purpose of this check? Could it be handled with an explicit prop?

Reply:
What I mean to ask was, why is there a comparison with a translated string? This is not a great pattern - translated strings are for display, and shouldn't be used in comparisons like this.

Reply:
Fixed by giving string comparision without using translated string and new variable (as we are using it only in one place)